prov/gni/src/gnix_datagram.c

```diff
@@ -273,6 +283,8 @@ int gnix_dgram_hndl_alloc(const struct gnix_fid_fabric *fabric,
 	struct gnix_dgram_hndl *the_hndl = NULL;
 	gni_return_t status;
 
+	GNIX_INFO(FI_LOG_EP_CTRL, "%s\n", __func__);
```

Comment:
Not sure what you're trying to do with this statement, but if I understand it correctly you just want to print out the function name that it has entered. You might want to consider: 
```
GNIX_TRACE(FI_LOG_EP_CTRL, "\n");
```
GNIX_TRACE already prints the `__func__` in the expanded output. It will also be visible at both `FI_LOG_LEVEL="trace"` and `FI_LOG_LEVEL="info"`